gateways/checkoutdotcom/request_builder.go

```diff
@@ -0,0 +1,59 @@
+package checkoutdotcom
+
+import (
+	"github.com/BoltApp/sleet"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/payments"
+)
+
+func buildChargeParams(authRequest *sleet.AuthorizationRequest) *payments.Request {
```

Comment:
good to follow the pattern of returning error.

Reply:
Added